Networking/Networking/Remote/LeaderboardsRemote.swift

```diff
@@ -30,7 +30,12 @@ public class LeaderboardsRemote: Remote {
             ParameterKeys.forceRefresh: forceRefresh
         ]
 
-        let request = JetpackRequest(wooApiVersion: .wcAnalytics, method: .get, siteID: siteID, path: Constants.path, parameters: parameters)
+        let request = JetpackRequest(wooApiVersion: .wcAnalytics,
+                                     method: .get,
+                                     siteID: siteID,
+                                     path: Constants.path,
+                                     parameters: parameters,
+                                     availableAsRESTRequest: true)
```

Comment:
any reasons we don't allow `availableAsRESTRequest: true` for the deprecated leaderboards request `loadLeaderboardsDeprecated` below for sites with WC version under 6.7? is it because application passwords are supported for higher WC version?